data/scenarios/Testing/1575-structure-recognizer/1575-bounding-box-overlap.yaml

```diff
@@ -15,7 +15,7 @@ objectives:
         Build 3 `chevron`{=structure} structures
     condition: |
       def length : (rec l. Unit + a * l) -> Int = \l.
-        case l (\_. 0) (\cons. 1 + length (snd cons))
+        case l (\_. 0) (\cons. match cons \_. \tl. 1 + length tl)
```

Comment:
This is one of the few cases where snd or the flip version would look better. Operator section would be a cool trick.

Reply:
How about if you spell a keyword backwards, it turns into a flipped version of itself?
```
case l (\_. 0) (hctam \_. \tl. 1 + length tl)
```
:wink:

Reply:
That would be so whimsical and fit the theme, I actually love it. 😆